.changesets/config_nathanmarcos_redis_cache_enabled.md

```diff
@@ -0,0 +1,27 @@
+### Configuration param to disable Redis cache for APQ and Query Planning ([Issue #3506](https://github.com/apollographql/router/issues/3506))
+
+The distributed [Query Plan](https://www.apollographql.com/docs/router/configuration/distributed-caching#distributed-query-plan-caching) and [APQ](https://www.apollographql.com/docs/router/configuration/distributed-caching#distributed-apq-caching) caching are [enterprise features](https://www.apollographql.com/docs/router/enterprise-features/) and require a [graph API key](https://www.apollographql.com/docs/graphos/api-keys/#graph-api-keys). In development environment, you might not have such a key and would need to manually change Router's configuration file to prevent a license violation error at startup. Now you can conveniently disable it by using an environment variable:
```

Comment:
```suggestion
Both [distributed query plan caching](https://www.apollographql.com/docs/router/configuration/distributed-caching#distributed-query-plan-caching) and [distributed APQ caching](https://www.apollographql.com/docs/router/configuration/distributed-caching#distributed-apq-caching) are [GraphOS Enterprise features](https://www.apollographql.com/docs/router/enterprise-features/) and require a [graph API key](https://www.apollographql.com/docs/graphos/api-keys/#graph-api-keys). With previous releases, if you didn't have a graph API key (possibly in a development environment), you had to manually change the router's configuration file to disable the features in order to prevent a license violation error at startup. Now in this release, you can conveniently disable the features by using an environment variable, for example with `ENABLE_REDIS_CACHE`:
```

Reply:
Thanks for the review @shorgi . Unfortunately it seems that this PR won't move forward.